docs/feature-gates.md

```diff
@@ -250,8 +245,24 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit. Node may lose network connection when `antrea-agent`
+or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
+`AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
+All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
```

Comment:
An empty line is needed if you want two paragraphs.

Reply:
Thanks. Fixed.